containers/datalab/content/run-debug.sh

```diff
@@ -15,21 +15,25 @@
 
 # Runs the docker container locally.
 
-export DATALAB_ENV=debug
+# We use the presence of /root/.config to tell whether to show landing page.
```

Comment:
As I know run-debug.sh is a mode between local managed VM run and real managed VM run, which is supposed to debugging managed VM Datalab. We probably don't need it here.